src/main/java/teammates/ui/controller/InstructorCourseEnrollSaveAction.java

```diff
@@ -105,23 +107,19 @@ public int compare(StudentAttributes o1, StudentAttributes o2) {
     @SuppressWarnings("unchecked")
```

Comment:
Maybe you can remove the precondition in the javadoc comments, since it's not needed anymore. Other than that, looks good